README.md

```diff
@@ -21,15 +21,24 @@ Identity::Hostdata.domain
 # => "login.gov"
 ```
 
-Set configs from YML files in S3
+Set configs from YML files in S3 or Secrets Manager
 
 ```ruby
 Identity::Hostdata.load_config!(
   app_root: Rails.root,
   rails_env: Rails.env
 ) do |builder|
+  # YML
   builder.add(:some_option, type: :string)
   builder.add(:other_option, type: :json)
+
+  # SSM
+  builder.add(:prop_name, secrets_manager_name: 'secrets-manager-name', type: :string)
```

Comment:
one issue with this, that I think @mitchellhenke had some ideas for also, is going to be per-env overrides, especially in local development.

In local dev, we don't have a SecretsManager instance we can connect to, so I'm going to need some syntax to override this.

We can probably leverage `Identity::Hostadata.in_datacenter?` somehow

Reply:
I was getting at that a bit here because just instantiating is enough to make an AWS call: https://github.com/18F/identity-hostdata/compare/mitchellhenke/secrets-manager?expand=1#diff-140e6df2cf49cbef69cccfba89b6a0094b6478a832ec8c319502f51010f3239fR126

Reply:
Added a version in 325df6f607371ef651106116b001502f12ec997d

Reply:
Does this support different keys per-environment? e.g. I want to fetch `dev/idp/database_password` from secrets manager for `dev`, and `mhenke/idp/database_password` for `mhenke`

Reply:
not currently, but we can always add that later. this will let us develop today with some breaking changes for the data warehouse